packages/synthetic-chain/src/cli/proposals.ts

```diff
@@ -51,9 +51,19 @@ export function readProposals(proposalsParent: string): ProposalInfo[] {
   const proposalsDir = path.join(proposalsParent, 'proposals');
   const proposalPaths = fs
     .readdirSync(proposalsDir, { withFileTypes: true })
-    .filter(dirent => dirent.isDirectory()) // omit files
-    .map(dirent => dirent.name)
-    .filter(name => name.includes(':')); // omit node_modules
+    .filter(dirent => {
+      const hasPackageJson = fs.existsSync(
+        path.join(dirent.path, dirent.name, 'package.json'),
```

Comment:
hooray!  This has bitten me more than once.

Reply:
Thanks for pointing it out!